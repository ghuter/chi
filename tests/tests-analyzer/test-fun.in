puts :: fun (str: ^u8) i32;

main :: fun (argc: i32, argv: ^^u8) i32 {
	puts("Hello worlds!");
}
