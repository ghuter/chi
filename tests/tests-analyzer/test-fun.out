SFUN(main, argc : i32, argv : ^^u8, {SCALL(puts, ECSTS(Hello worlds!))}) : i32
